src/ui/utils/devtools-toolbox.js

```diff
@@ -54,11 +54,14 @@ export class DevToolsToolbox {
 
     const panels = {
       debugger: DebuggerPanel,
-      inspector: InspectorPanel,
+      inspector: Inspector,
```

Comment:
could we also remove this Inspector as well?

Reply:
the entire inspector.ts file?